File: Beginner/HOW MANY DIGITS DO I HAVE (HOWMANY)/digits.java
import java.util.*;
import java.lang.*;
import java.io.*;


class Codechef {
	public static void main (String[] args) throws java.lang.Exception {
		Scanner scn = new Scanner(System.in);
		if(scn.hasNext()){
		    int num=scn.nextInt();
		    int d = 0;
		    while(num!=0){
		        d++;
		        num /= 10;
		    }
		    if(d==1)
		    System.out.println("1");
		    else if(d==2)
		    System.out.println("2");
		    else if(d==3)
		    System.out.println("3");
		    else if(d>3)
		    System.out.println("More than 3 digits");
		}
	}
}

File: Beginner/HOW MANY DIGITS DO I HAVE (HOWMANY)/digits.cpp
#include <iostream>
using namespace std;

int main() {

    int n, d=0;
    cin>>n;

    while(n!=0){
        n/=10;
        d++;  
    }

    if(d==1) {
        cout<<"1";
    }
    else if(d==2) {
        cout<<"2";
    }
    else if(d==3) {
        cout<<"3";
    }
    else {
        cout<<"More than 3 digits";
    }


    return 0;
}
